packages/tupaia-web/src/features/DashboardItem/EnlargedDashboardItem.tsx

```diff
@@ -5,15 +5,17 @@
 
 import React from 'react';
 import styled from 'styled-components';
-import { useParams, useSearchParams } from 'react-router-dom';
+import { Link, useLocation, useParams, useSearchParams } from 'react-router-dom';
+import { KeyboardArrowLeft } from '@material-ui/icons';
 import { Typography } from '@material-ui/core';
-import { FlexColumn } from '@tupaia/ui-components';
+import { FlexColumn, IconButton } from '@tupaia/ui-components';
 import { URL_SEARCH_PARAMS } from '../../constants';
 import { useDashboards } from '../../api/queries';
 import { DashboardItemContent } from './DashboardItemContent';
 import { useDateRanges } from '../../utils';
 import { useReport } from '../../api/queries/useReport';
 import { DateRangePicker, Modal } from '../../components';
+import { DashboardItemType } from '../../types';
 
 const Wrapper = styled.div<{
```

Comment:
I'm not sure exactly where it needs to go but can we please add a min-width for the enlarged dashboard items somewhere to match the OG site. When the Matrix only has one row or is loading it shrinks down.